contracts/CTKNCrowdsale.sol

```diff
@@ -11,10 +11,34 @@ import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
 contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale {
```

Comment:
Do we have to use IndividuallyCappedCrowdsale to limit / cap individual amounts or is there no such restriction? @harrisonperl

Reply:
Currently this only puts a total cap, but not on a per investor basis

Reply:
If so perhaps we can refactor after this PR is finished since we don't want to change too much at once

Reply:
Hard to tell if it will be needed: https://www.seedinvest.com/blog/regulation-a-equity-crowdfunding-rules 

"Investment Limits: For Tier II, individual investors can invest a maximum of the greater of 10% of their net worth or 10% of their net income in a Reg A+ offering (per offering). There are no investment limits under Tier 1."

Basically in KYC we can get this data, but I don't think we'll have anyone putting 10% of their net worth on this, either way it doesn't translate to an individual cap

Reply:
There are no individual caps.